airgun/views/host.py

```diff
@@ -774,7 +786,6 @@ class HostsTaxonomyMismatchRadioGroup(GenericLocatorWidget):
         </form>
     """
 
-    taxonomy = None
```

Comment:
The CI checks was failing because of this. I don't know the reason but CI checks started passing after I removed this.

Reply:
Not familar with this, do you remember the CI check that was failing here? This is only present exactly in this widget, no other place in Airgun.

Reply:
It is failing in make docs with error 
```
Warning, treated as error:
airgun/docs/autoapi/airgun/views/host/index.rst:1845:duplicate object description of airgun.views.host.HostsTaxonomyMismatchRadioGroup.taxonomy, other instance in autoapi/airgun/views/host/index, use :no-index: for one of them

```